fastlane/spec/actions_specs/set_github_release_spec.rb

```diff
@@ -4,7 +4,7 @@
       it "sends and receives the right content from GitHub" do
         stub_request(:post, "https://api.github.com/repos/czechboy0/czechboy0.github.io/releases").
           with(body: "{\"tag_name\":\"tag33\",\"name\":\"Awesome Release\",\"body\":\"Bunch of new things :+1:\",\"draft\":false,\"prerelease\":false,\"target_commitish\":\"test\"}",
-          headers: { 'Authorization' => 'Basic MTIzNDVhYmNkZQ==', 'Host' => 'api.github.com:443', 'User-Agent' => 'fastlane-set_github_release' }).
+          headers: { 'Authorization' => 'Basic MTIzNDVhYmNkZQ==', 'Host' => 'api.github.com:443', 'User-Agent' => 'fastlane-github_api' }).
```

Comment:
No its a very good point, especially if github were to throttle requests based on the header, I've just made some changes to the tests to show you can override it on the `GithubApiAction`. So if you use the optional `headers:` parameter, you can set `User-Agent` to anything you want. By default it will set to `github_api`. I guess if people find the need we can pass `headers` along from any source github api action as a simple future change.